.travis.yml

```diff
@@ -8,6 +8,7 @@ env:
  global:
    - PACKAGE="uri"
    - POST_INSTALL_HOOK="opam --yes depext -yui react ssl lwt"
+   - PINS="uri-sexp:."
```

Comment:
below, we should run at least one CI job that attempts to compile + test `uri-sexp` (see e.g. https://github.com/mirage/ocaml-cstruct/blob/8d7ab7caf5a5a6b419f75cbbded78ee751a6e016/.travis.yml#L13 - it's basically `PACKAGE="uri-sexp" ...`)